game/scripts/vscripts/Sections/Chapter6/SectionClosing.ts

```diff
@@ -183,6 +184,8 @@ function onStart(complete: () => void) {
                 tg.audioDialog(LocalizationKey.Script_6_Closing_2, LocalizationKey.Script_6_Closing_2, sunsFan),
                 tg.audioDialog(LocalizationKey.Script_6_Closing_3, LocalizationKey.Script_6_Closing_3, slacks),
                 tg.audioDialog(LocalizationKey.Script_6_Closing_4, LocalizationKey.Script_6_Closing_4, sunsFan),
+                tg.audioDialog(LocalizationKey.Script_6_Closing_5, LocalizationKey.Script_6_Closing_5, sunsFan),
```

Comment:
This needs to be done after https://github.com/ModDota/dota-tutorial/pull/410 is merged, and probably shouldn't be done here (conflicts for sure).